tycho-common/src/storage.rs

```diff
@@ -478,6 +478,23 @@ pub trait ProtocolGateway {
         chain: &Chain,
         pagination_params: Option<&PaginationParams>,
     ) -> Result<WithTotal<Vec<String>>, StorageError>;
+
+    /// Retrieve the total value locked for a component
+    ///
+    /// Fetches the total value locked for a component
+    ///
+    /// # Parameters
+    /// - `chain` The chain for which to retrieve the total value locked
+    /// - `system` The protocol system for which to retrieve the total value locked
+    /// - `ids` The ids of the components to retrieve the total value locked for
+    ///
+    /// # Return
+    /// The total value locked for the component
```

Comment:
More accurate would be something more like:
```suggestion
    /// A result with a map of component ids to their TVL. Err if storage access failed.
```